apps/vs-code-designer/src/app/commands/dataMapper/DataMapperExt.ts

```diff
@@ -52,20 +40,20 @@ export default class DataMapperExt {
       }
     );
 
-    DataMapperExt.panelManagers[dataMapName] = new DataMapperPanel(panel, dataMapName);
-    DataMapperExt.panelManagers[dataMapName].panel.iconPath = {
-      light: Uri.file(path.join(DataMapperExt.context.extensionPath, 'assets', 'wand-light.png')),
-      dark: Uri.file(path.join(DataMapperExt.context.extensionPath, 'assets', 'wand-dark.png')),
+    ext.dataMapPanelManagers[dataMapName] = new DataMapperPanel(panel, dataMapName);
+    ext.dataMapPanelManagers[dataMapName].panel.iconPath = {
+      light: Uri.file(path.join(ext.context.extensionPath, 'assets', 'wand-light.png')),
+      dark: Uri.file(path.join(ext.context.extensionPath, 'assets', 'wand-dark.png')),
     };
-    DataMapperExt.panelManagers[dataMapName].updateWebviewPanelTitle();
-    DataMapperExt.panelManagers[dataMapName].mapDefinitionData = mapDefinitionData;
+    ext.dataMapPanelManagers[dataMapName].updateWebviewPanelTitle();
+    ext.dataMapPanelManagers[dataMapName].mapDefinitionData = mapDefinitionData;
 
     // From here, VSIX will handle any other initial-load-time events once receive webviewLoaded msg
   }
 
   public static log(text: string) {
-    DataMapperExt.outputChannel.appendLine(text);
-    DataMapperExt.outputChannel.show();
+    ext.outputChannel.appendLine(text);
+    ext.outputChannel.show();
```

Comment:
NIT: I have notice that the output channel opens everytime the extension gets activated. Its not a blocker but just wanted to point out if we still want to show it

Reply:
Would you say we should not show the outputChannel when we are logging?

I noticed in designer, we tend to use `ext.outputChannel.appendLog(message)` ineatead. Should I remove this log function as transition towards that? Not exactly sure if we should keep those `log`, `showWarning` and `showError` functions.